src/styles/components/responsive.scss

```diff
@@ -12,48 +14,48 @@
 
 .hidden-desktop {
     @include desktop {
-        display: none;
+        display: none !important;
     }
 }
 
 .hidden-tablet {
     @include tablet {
-        display: none;
+        display: none !important;
     }
 }
 
 .hidden-phone {
     @include phone {
-        display: none;
+        display: none !important;
     }
 }
 
 .visible-desktop {
     @include phone {
-        display: none;
+        display: none !important;
```

Comment:
You have `.visible-desktop` defined exactly the same as `.hidden-desktop`, with `display: none`. Is that right? (Similarly the other visibles and hiddens.)